website/src/data/navigation.yaml

```diff
@@ -76,6 +76,8 @@
       path: /api-reference/create-mc-app
     - title: Custom Webpack Config
       path: /api-reference/custom-webpack-config
+    - title: Deployment previews
+      path: /api-reference/deployment-previews
```

Comment:
Hmm I'm not sure this should go into the API Reference, maybe into the Concepts section.

Reply:
Sure. I picked API Reference from the docs link in the feature preview on Staging Merchant Center.